src/site/layouts/va_form.drupal.liquid

```diff
@@ -50,13 +50,18 @@
             </dd>
           </div>
 
-          {% if fieldVaFormRevisionDate %}
+          {% if fieldVaFormRevisionDate or fieldVaFormIssueDate %}
             <div>
               <dt class="vads-u-font-weight--bold vads-u-display--inline">
                 Form last updated:
               </dt>
               <dd class="vads-u-display--inline">
-                {{ fieldVaFormRevisionDate.value | humanizeDate }}
+                {% assign revisionDateIsLatestUpdate = fieldVaFormRevisionDate.value | isLaterThan: fieldVaFormIssueDate.value %}
```

Comment:
you can't use a liquid filter in an if-statement so i had to assign it to a var beforehand